### PR TITLE
Replace loader class with Spinner

### PR DESCRIPTION
**Pull request by vilsbole**
- Refactor Spinner to SC component
- Replace `.loading` with Spinner
- Prepare to reuse `AppLoading`

### CODE REVIEW

File: src/components/AppLoading.tsx [hunk 1]
```diff
@@ -47,14 +32,14 @@ const LoadingSubText = styled.h6`
...
-    <Layout>
-      <DashboardPanelLoading>
-        <Loader className="loading" />
-        <LoadingText>{translateRaw('APP_LOADING')}</LoadingText>
-        <LoadingSubText>{translateRaw('APP_LOADING_SUBHEADER')}</LoadingSubText>
-      </DashboardPanelLoading>
-    </Layout>
-  );
-};
+const AppLoading = () => (
```
Comment (FrederikBolding) [inline src/components/AppLoading.tsx:35]: This component now looks wrong, it has a very low width. It's difficult to take a screenshot of, but is clear if you go to a page that you haven't already loaded.

File: src/components/Spinner.tsx [hunk 2]
```diff
@@ -1,20 +1,52 @@
...
+   100% { transform: rotate(360deg)}
+`;
 
-interface SpinnerProps {
-  size?: Size;
-  light?: boolean;
-}
+const Svg = styled.svg<{ $size: number } & SpaceProps>`
+  ${space}
+  animation: ${rotate} 0.8s linear infinite;
```
Comment (FrederikBolding) [inline src/components/Spinner.tsx:12]: Why a different duration?

File: src/components/WalletUnlock/DeterministicAccountTable.tsx [hunk 3]
```diff
@@ -321,100 +305,108 @@ const DeterministicTable = ({
...
-                      <Trans id="DETERMINISTIC_GENERATE_FRESH_ADDRESS" />
-                    </STypography>
-                  )}
-            </GenerateAddressButton>
-          </Body>
-        )}
+              </LinkContainer>
+            </Row>
+          ))}
+          <GenerateAddressButton
```
Comment (FrederikBolding) [inline src/components/WalletUnlock/DeterministicAccountTable.tsx:377]: What has changed here at the bottom? Just whitespace? The diff is hard to read 😆

**Review by FrederikBolding — CHANGES_REQUESTED** [review]

File: src/components/WalletUnlock/DeterministicAccountTable.tsx (hunk 3 above)
Comment (vilsbole) [reply src/components/WalletUnlock/DeterministicAccountTable.tsx:377]: Prettified :)

File: src/components/Spinner.tsx (hunk 2 above)
Comment (vilsbole) [reply src/components/Spinner.tsx:12]: median between the two previous values.

File: src/components/AppLoading.tsx (hunk 1 above)
Comment (vilsbole) [reply src/components/AppLoading.tsx:35]: Solved with `width: 100%`

**Review by FrederikBolding — APPROVED** [review]
Good to merge when tests pass

**Pull request merged**